tsconfig.json

```diff
@@ -1,24 +1,32 @@
 {
   "compilerOptions": {
+    "target": "ES2020",
```

Comment:
Nice catch. I updated `target` and `lib` to match TypeScript config we used in CKE5 repo.

Reply:
@filipsobol, can you explain why we can use optional chaining (and other) operators in this repo but cannot use them in the CKE5 repo even though both repos support es2019?

Reply:
In both the CKE5 repo and here we **could** use optional chaining, since in both environments our target is `ES2019`. With such a target, a simple `test?.prop1?.prop2` is transformed into `(_a = test === null || test === void 0 ? void 0 : test.prop1) === null || _a === void 0 ? void 0 : _a.prop2` (or similar, depending on the tooling), which works well in older environments like webpack 4, which we still support in the old installation methods.

However, in CKE5 the code is transformed BEFORE the tests are run, meaning that the code coverage is generated for the transformed code instead of simple `test?.prop1?.prop2`. This messes up the code coverage and generally makes it not worth using.

We don't have this problem in this repository, so we can use it just fine.